src/Components/CurrentWeather.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import partlycloudy from "../img/weather-icons/partlycloudy.svg";
+
+const CurrentWeather = () => {
+  return (
+    <div className="Todaysweather">
```

Comment:
Do these two nested divs (.Todaysweather and .currentWeather) have separate purposes?